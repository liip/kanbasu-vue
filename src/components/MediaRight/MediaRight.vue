<template>
  <div class="media__right">
    <slot></slot>
  </div>
</template>

<script>
export default {
  name: 'media-right',
};
</script>
