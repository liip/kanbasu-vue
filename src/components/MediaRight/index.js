import MediaRight from './MediaRight.vue';

MediaRight.install = function(Vue) {
  Vue.component(MediaRight.name, MediaRight);
};

export default MediaRight;
