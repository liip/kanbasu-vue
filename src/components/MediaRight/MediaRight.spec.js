import { shallowMount } from '@vue/test-utils';
import MediaRight from './MediaRight.vue';

describe('MediaRight.vue', () => {
  it('renders', () => {
    const wrapper = shallowMount(MediaRight, {
      slots: {
        default: 'slot',
      },
    });
    expect(wrapper).toMatchSnapshot();
  });
});
