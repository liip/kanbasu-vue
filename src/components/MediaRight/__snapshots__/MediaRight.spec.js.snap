// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MediaRight.vue renders 1`] = `<div class="media__right">slot</div>`;
